lib/elastic_apm/span.rb

```diff
@@ -75,6 +70,15 @@ def running?
       started? && !stopped?
     end
 
+    def stacktrace
```

Comment:
Why lazy load the prop instead? Perf or preference? 😊

Reply:
The goal was to move the `stacktrace_builder.build` call to a worker thread. I didn't test the performance difference though, just thought it made sense to do less work on the main thread.

Now that I'm looking at it again, it's probably better to keep an explicit `build_stacktrace` step rather than lazy load. That way it's easier to determine where the work actually happens.